metadata.json

```diff
@@ -83,6 +83,12 @@
       ]
     },
     {
+      "operatingsystem": "OpenSuSE",
```

Comment:
As per your other PR, I'm not sure this metadata is correct. I'll investigate some more.